### PR TITLE
Port sunpy.time to Python 3

### PR DESCRIPTION
**Pull request by Cadair**
This one was also easy, tests for sunpy.time are now passing under 2 and 3.

### CODE REVIEW

**Comment by Cadair** [comment]
@nabobalis cast your eye over this please.

File: sunpy/time/tests/test_julian.py [hunk 1]
```diff
@@ -10,10 +10,10 @@
 DATETIME_DATE_1 = datetime(1900, 1, 1, 12, 00, 00)
 STRING_DATE_1 = '1900/1/1 12:00:00'
 
-DATETIME_DATE_2 = datetime(1984, 11, 23, 14, 21, 05)
+DATETIME_DATE_2 = datetime(1984, 11, 23, 14, 21, 5)
```
Comment (nabobalis) [inline sunpy/time/tests/test_julian.py]: Why was this changed?

**Comment by nabobalis** [comment]
Is astropy.time not being used as well?

File: sunpy/time/tests/test_julian.py (hunk 1 above)
Comment (Cadair) [inline sunpy/time/tests/test_julian.py]: because numbers should not start with 0's. modernize converted them to octal.

**Comment by Cadair** [comment]
@nabobalis we don't really have a dep on astropy.time at the moment.

File: sunpy/time/tests/test_julian.py (hunk 1 above)
Comment (nabobalis) [inline sunpy/time/tests/test_julian.py]: Ah, that was a modernize change. Didn't think it would do that.

**Comment by nabobalis** [comment]
:+1:

File: sunpy/time/tests/test_julian.py [hunk 2]
```diff
@@ -10,10 +10,10 @@
...
 STRING_DATE_2 = '1984/11/23 14:21:05'
 
-DATETIME_DATE_3 = datetime(2174, 2, 11, 05, 02, 00)
+DATETIME_DATE_3 = datetime(2174, 2, 11, 5, 2, 00)
```
Comment (dpshelio) [inline sunpy/time/tests/test_julian.py]: the last 0 should be then one 0 too.

**Comment by Cadair** [comment]
@dpshelio ping

File: continuous-integration/python3.sh [hunk 3]
```diff
@@ -6,3 +6,5 @@ python setup.py install
 python -c "import sunpy.data"
 python -c "import sunpy.data; sunpy.data.download_sample_data()"
 python -c "import sunpy.data.sample"
+
+python setup.py test -P time
```
Comment (dpshelio) [inline continuous-integration/python3.sh]: sometimes I've found that if a script does not ends in a new line such line does not work...

**Comment by dpshelio** [comment]
:+1:
